e2e/test/provisioning/ProvisioningServiceClientE2ETests.cs

```diff
@@ -159,21 +159,23 @@ public async Task ProvisioningServiceClient_GetIndividualEnrollmentAttestation(A
             using var provisioningServiceClient = new ProvisioningServiceClient(TestConfiguration.Provisioning.ConnectionString);
             string registrationId = AttestationTypeToString(attestationType) + "-" + Guid.NewGuid();
 
-            IndividualEnrollment individualEnrollment = await CreateIndividualEnrollmentAsync(
-                    provisioningServiceClient,
-                    registrationId,
-                    attestationType,
-                    null,
-                    null,
-                    AllocationPolicy.Static,
-                    null,
-                    null,
-                    null,
-                    Logger)
-                .ConfigureAwait(false);
+            IndividualEnrollment individualEnrollment = null;
 
             try
             {
+                individualEnrollment = await CreateIndividualEnrollmentAsync(
```

Comment:
Why is this moved into the try block? If this call fails, then there is nothing to cleanup, right?

Reply:
In some cases I found, the code that creates it has checks and may fail. So, we just want to be sure we clean it up.

Reply:
I checked this one, and it doesn't, but I think it is safe to keep it in here. We check for null, just in case.

Reply:
Adding the null check, because it wasn't actually done in this case.